react-common/components/util.tsx

```diff
@@ -46,7 +46,7 @@ export function fireClickOnEnter(e: React.KeyboardEvent<HTMLElement>) {
     }
 }
 
-export function classList(...classes: string[]) {
+export function classList(...classes: (string | undefined)[]) {
```

Comment:
What does allowing an entry to be undefined allow us to do?

Reply:
This was always the intended functionality of this method, but teachertool uses strict typing and so I had to make it explicit.

Reply:
The method filters out undefined elements.